plugins/woocommerce/README.md

```diff
@@ -13,6 +13,19 @@ This is the WooCommerce Core plugin. Here you can browse the source and keep tra
 
 If you are not a developer, please use the [WooCommerce plugin page](https://wordpress.org/plugins/woocommerce/) on WordPress.org.
 
+## Getting Started
+
+Please make sure you follow the [repository's getting started guide](../../README.md#getting-started) first!
+
+```bash
+# Make sure that WooCommerce Core and all of its dependencies are built
+pnpm -- turbo run build --filter=woocommerce
+# Start the development environment
+pnpm -- wp-env start
+```
+
+You should now be able to visit http://docker.local:8888/ and access WooCommerce environment.
+
```

Comment:
Thanks for adding this! 

Is there a way to rebuild just the assets or watch for changes (either globally or in the plugins/woocommerce subdirectory) to get automatic rebuild?

Reply:
I left out anything that I felt was too in flux to document until it's ready. We're going to be renaming the `woocommerce-legacy-assets` and `@woocommerce/admin-library` packages, after which I'll add documentation here about the fact those two components are part of the plugin and can be built independently.

As for `watch` functionality, that's a bit more complicated. Turborepo doesn't outright support it. I'd like for us to add a `"dev"` turbo command that kicks off any related watching functionality in any specific repository. For now you can use `pnpm {command} --filter={package}` to run whatever watch command a project provides. This is pretty project-specific though.

For your own information, you can rebuild the assets using `pnpm -- turbo run build --filter=woocommerce-legacy-assets` right now.